ballot/app/src/store.js

```diff
@@ -0,0 +1,92 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+import oasis from '@oasislabs/client';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    args: [
+      'Which starter Pokemon is the best?',
+      [
+        'Bulbasaur',
+        'Charmander',
+        'Squirtle',
+      ],
+    ],
+    ballot: null,
+    bytecode: '/assets/bytecode/ballot.wasm',
+    gateway: 'ws://localhost:8546',
+    mnemonic: 'range drive remove bleak mule satisfy mandate east lion minimum unfold ready',
```

Comment:
please make a note here that says that people copying this template should actually do the service interaction using the dev gateway and that the client does signing locally using in-browser deoxysii key